src/private/meeting.ts

```diff
@@ -1,81 +1,8 @@
 import { ensureInitialized, sendMessageRequestToParent } from '../internal/internalAPIs';
 import { GlobalVars } from '../internal/globalVars';
+import { MeetingRoomCapability, MeetingRoomInfo, MeetingRoomState } from './interfaces';
 
 export namespace meeting {
-  /**
-   * @private
-   * Hide from docs
-   *
-   * Data structure to represent a meeting room.
-   */
-  export interface MeetingRoomInfo {
```

Comment:
Why move these out of the meeting namespace?

Reply:
I noticed there is a "interfaces.ts" that contains a sort of interfaces declarations. I was thinking that was a existing pattern.
Can revert this if it is not good.

Reply:
Has moved the meeting related interfaces back to meeting namespace.